packages/experiments/src/components/TilesList/TilesList.types.ts

```diff
@@ -123,4 +124,13 @@ export interface ITilesListProps<TItem> extends IBaseProps, React.Props<TilesLis
    * Callback for when the active element within the list's FocusZone changes.
    */
   onActiveElementChanged?: (element: HTMLElement) => void;
+  /**
+   * calback to return the height of a single item in pixels
+   */
+  getItemHeight?: (index: number) => number;
```

Comment:
It seems like `getItemHeight` would conflict with `minRowHeight` and `desiredSize`. What item does this apply to?

Reply:
I removed this callback prop in the latest iteration.

Reply:
"Item" is a "cell" in this case.  This should be made clear in the latest revision.